backend/backend/settings.py

```diff
@@ -26,7 +27,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
```

Comment:
@Mansiviramgama @Mayank-0203 This means, I have allowed all the hosts to access my application...

Say, I'm running my application on localhost, it will surely work... but when I run my app on my local network, i.e. with ip addr 192.168.1.103:8000 then my django app will throw an error... So for fixing that issue I have added this config.